stock_available_to_promise_release/models/stock_move.py

```diff
@@ -371,6 +371,10 @@ def _after_release_update_chain(self):
             moves = moves.mapped("move_orig_ids")
         pickings = self.env["stock.picking"].browse(picking_ids)
         pickings._after_release_update_chain()
+        # Set the highest priority on all pickings in the chain
+        highest_priorities = pickings.mapped("priority")
```

Comment:
```suggestion
        priorities = pickings.mapped("priority")
```
The highest is then given by the call to `max`, not before :p

Reply:
True, renamed.

Reply:
It remains one place where it should be renamed ;)

Reply:
Should be good now.